examples/cat_hs/exec/cat_hs/BUILD.bazel

```diff
@@ -0,0 +1,18 @@
+load(
+    "@rules_haskell//haskell:defs.bzl",
+    "haskell_binary",
+)
+load(
+    "@rules_haskell//haskell:cabal.bzl",
+    "haskell_cabal_library",
```

Comment:
Why import `haskell_cabal_library` if this file doesn't use it?

Reply:
Redundant.

Reply:
I can do a PR!